images/metering-ansible-operator/roles/meteringconfig/defaults/main.yml

```diff
@@ -148,7 +148,7 @@ _meteringconfig_presto_client_key: ""
 _meteringconfig_tls_root_ca_certificate: ""
 _meteringconfig_tls_root_ca_key: ""
 
-meteringconfig_root_ca_overrides:
+metering_root_ca_overrides:
```

Comment:
Maybe just `_meteringconfig_root_ca_overrides`?

Reply:
I believe when I initially tried that, I was having a hard time getting that to work. 

I'll try to use this again though.

Reply:
Update: seems like that's not the case, and this works as expected.